CMP1903_A1_2324/Testing.cs

```diff
@@ -5,7 +5,52 @@ namespace Dice2
 {
 	public class Testing
 	{
-		
+		public Testing()
+		{
+		}
+
+		//Tests that the Die runs
+        internal void Run()
+        {
+			CreateDie();
+			TestRollDie();
+        }
+
+		//Checks whether the Die is null or not
+		public void CreateDie()
+		{
+			Die testdie = new Die();
+			if(testdie == null)
```

Comment:
In accordance to C# Google Style Guide Sheet, after 'if' there should be a space.